pages/message-board.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="description"
+      content="Regis Altare themed booster pack opening game. Open packs and collect all the cards online in this free web app."
+    />
+    <meta
+      name="keywords"
+      content="regis,altare,holopro,holostars,booster,pack,box,simulator,sim,open,opener,opening,tcg,set,card,cards,free,online,pulls,pack,packs,breaks,app,games,game,"
+    />
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:title" content="Regis Altare Booster Pack Simulator" />
+    <meta
+      name="twitter:description"
+      content="Regis Altare themed booster pack opening game. Open packs and collect all the cards online in this free web app."
+    />
+    <!-- Enables media queries on mobile -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!-- CSS-->
+    <link type="text/css" rel="stylesheet" href="../css/main.css" />
+
+    <title>Regis Altare Booster Pack Simulator</title>
+  </head>
+
+  <body>
+    <main>
+      <h1>The Regis Altare Birthday Project 2023</h1>
+
+      <nav>
+        <a href="../index.html">Collection</a>
+        <a class="active" href="message-board.html">Birthday Message Board</a>
+      </nav>
+
+      <section>
+        <a href="https://www.kudoboard.com/boards/qQ8344eM"
+          >Go to message wall</a
+        >
+        <video autoplay muted loop controls>
+          <source
+            src="https://res.cloudinary.com/dazcxdgiy/video/upload/v1669320701/kudoboard_qbu1gi.mp4"
+            type="video/mp4"
+          />
+          Your browser does not support the video tag.
+        </video>
+      </section>
```

Comment:
rest of file is copy paste from index.html. Here are the main changes

Reply:
I made a new pages folder since that's what the repo we're referencing did, but tbh remembering to update the file paths was a bit tedious. If anyone prefers having all the html pages be on the same level as index.html I can change it.

Reply:
I feel that having the pages in a designated folder is better for organization, to have everything in its place instead of cluttering the root, so I'm fine with it. And yeah, we don't have many pages so hardcoding the class should be fine. If it turns out the project needs way more pages for some reason we can always change it to dynamic styling.